source/guides/clear/autospec.rst

```diff
@@ -326,6 +326,90 @@ and dependencies so autospec can complete a successful build.
    if autospec is not able to automatically find the correct dependency on its
    own.
 
+.. TODO: Document how to set up a license server for use with autospec
+
+Example 5: Control file management
+**********************************
+
+TBD
+
+Example 6: Update an existing package
+*************************************
+
+Generally, developers can expect a more streamlined approach to managing a large collection of packages with :command:`autospec`.
+
+The |CL| team prefers to carry no patches and seeks to make the latest releases work. If we do need patches, we use :command:`autospec` to add, remove, or manage patches. The :command:`autospec` control files are integral to the patch management process.
+
+Adding and submitting patches
+=============================
+
+* To add patches to |CL| upstream, follow our process
+  in the |CL| `distribution repo contributing guide`_.
+
+* To submit a patch to upstream, follow our process in
+  `contributing new packages`_.
+
+If you maintain a downstream derivative of |CL| and you want to integrate the patch into your mix, follow the process in :ref:`mixer`.
+
+Assuming you have followed the above process, :command:`autospec` has generated a new spec file. View the changes by following the next section.
+
+Refresh a package and inspect
+-----------------------------
+
+#. To refresh the spec package and recreate RPM files, navigate to its
+   directory. In this case we use :command:`m4`.
+
+#. Navigate to the top-level directory of the workspace
+
+   .. code-block:: bash
+
+      cd clearlinux
+
+   - where clearlinux is the top level of the tooling workspace
```

Comment:
I think `clearlinux` should be emphasized in the markup on the above line, however you do that in RST :-)